### ISSUE TITLE
diskquota stop working after removing any extension

### ISSUE THREAD

**Issue by RekGRpth** [greenplum-db/diskquota#377]
```sql
CREATE EXTENSION diskquota;
CREATE EXTENSION
CREATE EXTENSION gp_inject_fault;
CREATE EXTENSION
SELECT diskquota.init_table_size_table();
 init_table_size_table 
-----------------------
 
(1 row)

SELECT diskquota.set_schema_quota(current_schema, '1MB');
 set_schema_quota 
------------------
 
(1 row)

SELECT diskquota.status();
              status              
----------------------------------
 ("soft limits",on)
 ("hard limits",off)
 ("current binary version",2.2.1)
 ("current schema version",2.2)
(4 rows)

DROP EXTENSION gp_inject_fault;
DROP EXTENSION
SELECT diskquota.status();
              status              
----------------------------------
 ("soft limits",paused)
 ("hard limits",off)
 ("current binary version",2.2.1)
 ("current schema version",2.2)
(4 rows)

SELECT diskquota.resume();
 resume 
--------
 
(1 row)

SELECT diskquota.status();
              status              
----------------------------------
 ("soft limits",paused)
 ("hard limits",off)
 ("current binary version",2.2.1)
 ("current schema version",2.2)
(4 rows)
```
in log
```csv
2023-09-12 21:23:14.502529 +05,"postgres","postgres",p548094,th-680920960,"127.0.0.1","34610",2023-09-12 21:23:13 +05,0,con16,cmd27,seg-1,,,,sx1,"LOG","00000","statement: DROP EXTENSION gp_inject_fault;",,,,,,"DROP EXTENSION gp_inject_fault;",0,,"postgres.c",1668,
2023-09-12 21:23:14.504175 +05,,,p547808,th-680920960,,,,0,con4,cmd16,seg-1,,,,,"LOG","00000","[diskquota launcher]: received create/drop extension diskquota message, extension launcher",,,,,,,0,,"diskquota.c",1050,
2023-09-12 21:23:14.524626 +05,,,p548116,th-680920960,,,,0,con18,cmd5,seg-1,,,,,"LOG","00000","[diskquota] stop disk quota worker process to monitor database:postgres",,,,,,,0,,"diskquota.c",586,
```

**Comment by RekGRpth** [comment]
I came up with a workaround to run diskquota after it stopped working after removing any extension (until https://github.com/arenadata/diskquota/pull/25 was applied).

1) Execute the following sql script via psql in the database in which the diskquota has stopped working:
```sql
select current_database();
\gset

select oid from pg_database where datname = :'current_database';
\gset

\c diskquota
insert into diskquota_namespace.database_list select :oid::oid;
```
2) Restart the entire cluster

**Comment by zhrt123** [comment]
Hi @RekGRpth 
I found the code causing this problem. I will fix it in diskquota-2.3. Before this version, you can run the above script for a workaround.
